aiogram_dialog/widgets/kbd/datecalendar.py

```diff
@@ -0,0 +1,163 @@
+from datetime import date, datetime
+from time import mktime
+from calendar import monthcalendar
+from typing import List, Callable, Union, Awaitable
+
+from aiogram.types import InlineKeyboardButton, CallbackQuery
+
+from aiogram_dialog.dialog import Dialog
+from aiogram_dialog.manager.manager import DialogManager
+from aiogram_dialog.manager.intent import ChatEvent
+from aiogram_dialog.widgets.widget_event import WidgetEventProcessor, ensure_event_processor
+from .base import Keyboard
+
+OnDateSelected = Callable[[ChatEvent, "MonthCalendar", DialogManager, date], Awaitable]
+
+LEVEL = "LEVEL"
+YEARS_LEVEL = "Y"
+MONTHS_LEVEL = "M"
+YEAR_PREFIX = "YEAR:"
+MONTH_PREFIX = "MONTH:"
+NEXT_MONTH = "+"
```

Comment:
"+" is not next month! Next month is June. May be it is next month command or next month callback?